src/util/defaults.ml

```diff
@@ -152,6 +152,7 @@ let _ = ()
       ; reg Incremental "incremental.save"   "false" "Store incremental analysis results."
       ; reg Incremental "incremental.stable" "true"  "Reuse the stable set and selectively destabilize it (recommended)."
       ; reg Incremental "incremental.wpoint" "false" "Reuse the wpoint set (not recommended). Reusing the wpoint will combine existing results at previous widening points."
+      ; reg Incremental "incremental.reluctant" "true" "Destabilize nodes in changed functions reluctantly"
```

Comment:
I'm wondering if this should be enabled by default, given how its behavior can be quite unpredictable as is: it depends on the order of the functions destabilized reluctantly and may bypass reluctant destabilization as mentioned above.

Reply:
I turned it on as default because for my test runs on the _figlet_ repository the activation of reluctant destabilization did reduce the number of evaluations for quite a few commits but was never disadvantageous in this respect. But of course testing it on other repositories as well would be required for a better justified statement.